Update serverless-ai-tutorial.md (#900)
fermyon/developer: content/spin/serverless-ai-tutorial.md

{{ startTab "Rust"}}

> > Note: The following Rust source code, for this sentiment analysis example, uses serde. Please add the following dependencies to your Rust application's `Cargo.toml` file before you build/run the application.

Reviewer: The line that led into this tab block (line 229) and the section title (line 227) seemed to be leading directly into the source code.  With this note here the colon at the end looks amiss (though it works with the TypeScript tab).  I am not sure this will bother anybody else though!

> > Note: The following Rust source code, for this sentiment analysis example, uses serde. Please add the following dependencies to your Rust application's `Cargo.toml` file before you build/run the application.

Reviewer: My usual hobby horse about cutting back the "notes" and the "pleases" and such like I'm afraid, just hopefully making it a bit crisper.

```suggestion
> The Rust source code for this sentiment analysis example uses the `serde` crate. Add the following dependencies to your Rust application's `Cargo.toml` file before you build/run the application:
```

and omg I got to suggest a colon to Tim

```toml

> serde = { version = "1.0", features = ["derive"] }

Reviewer: You could ask them to run:

```
$ cargo add serde -F derive
$ cargo add serde_json
```

This might provide a more specific "do this" and save them having to manually edit Cargo.toml?  I don't have strong feelings, just mentioning it as an idea.

Author: Thanks added this as an alternative.

serde_json = "1.0.85"
```

> The following Rust source code is an example of what goes in your `src/lib.rs` file:

Reviewer: I wouldn't describe this as an "example" - this is what we want them to write isn't it?

```suggestion
Change your `lib.rs` file to read as follows:
```

(or better phrasing, I'm not mad keen on 'read' here)

- b) have configured your `spin.toml` file to explicitly configure the model (as shown above), and
- c) have [signed up for the Serverless AI private beta](https://developer.fermyon.com/cloud/serverless-ai).

> Now, let's build and run our Spin Application locally. (***Note:** If you are following along with the TypeScript/JavaScript example, you will first need to run `npm install`. Otherwise, please continue to the following `spin` command.):

Reviewer: It seems like the `npm install` instruction might fit in the same position as the `cargo add` instruction?  Not sure if other packages get added later though.

If it has to be here, should we have tabs instead of bulking up this paragraph with a "Note:"?

> Now, let's build and run our Spin Application locally. (***Note:** If you are following along with the TypeScript/JavaScript example, you will first need to run `npm install`. Otherwise, please continue to the following `spin` command.):

Reviewer: if you keep it as is

```suggestion
Now, let's build and run our Spin Application locally. (**Note:** If you are following along with the TypeScript/JavaScript example, you will first need to run `npm install`. Otherwise, please continue to the following `spin` command.)
```

(I know about the preference for colons but it's not grammatical to have one after the end of a sentence and I will sustain mild injury on this hill.)
